composer.json

```diff
@@ -8,10 +8,10 @@
     "license": "MIT",
     "type": "project",
     "require": {
-        "php": ">=5.5.9",
-        "laravel/framework": "5.2.*",
+        "php": ">=5.6.4",
+        "laravel/framework": "5.3.*",
         "laravel/socialite": "^2.0",
-        "laravel/cashier": "~5.0",
+        "laravel/cashier": "~7.0",
```

Comment:
To my knowledge, cashier isn't even used anymore. Perhaps we can remove it as a dependency entirely?